modules/install-osp-create-network.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+:_content-type: CONCEPT
+[id="install-osp-create-network_{context}]
+= Creating dual-stack network for the cluster
+.Procedure
+. Create a network like following example:
++
+[source, terminal]
+----
+$ openstack network create
```

Comment:
I think we can omit the actual commands for network and subnet creation given they can be created in various different type, for example the user can provide a tenant network and also provider network.
Instead, we can just list the requirements around the network and subnet, for example:

-  IPv6 subnet connected to a router
- DHCP enabled on the subnet
-  list the address modes supported
- Details around the MTU value

Reply:
From a user perspective, can you confirm that the user will not require any assistance to know the syntax of commands? I specified as a reference for the users.
'

Reply:
I don't think that is needed.
My suggestion is to follow the same pattern of how the requirements were listed for a custom subnet: https://docs.openshift.com/container-platform/4.13/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-custom-subnet_installing-openstack-installer-custom

We could be a bit more specific though, by mentioning the specific fields of the IPv6 subnet command that must be set:

- ipv6-ra-mode can be set with: {dhcpv6-stateful,dhcpv6-stateless,slaac}
- ipv6-address-mode  can be set with: {dhcpv6-stateful,dhcpv6-stateless,slaac}
- dhcp must be enabled

All the other settings is up to the user to choose.

Here is the openstack doc of the subnet command:
https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/subnet.html